docs/api/middleware.rst

```diff
@@ -255,7 +256,20 @@ the responder method that the request would have been routed to. However, any
 
 In a similar manner, setting :attr:`~falcon.Response.complete` to ``True`` from
 within a *process_resource* method will short-circuit further request processing
-at that point.
+at that point. 
+
+In the example bellow, you can see how request processing will be short-circuit
```

Comment:
Suggested edit:
request processing will be short-circuited once ``resp.complete`` has been set to ``True``, i.e., the framework will prevent `mob3.process_request`, all `process_resource` methods, as well as the routed responder method from processing the request.